File: MPI/mpicheck/tests/functions/1.0/MPI_Error_string.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       CHARACTER*(10) var_1
       INTEGER var_2
       INTEGER var_3
        call mpi_error_string(var_0, var_1, var_2, var_3)
       call pmpi_error_string(var_0, var_1, var_2, var_3)
        end program main
    

File: MPI/mpicheck/tests/functions/1.0/MPI_Error_string.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    int var_0;
    char *var_1;
    int *var_2;
    int ret;
    /* calls */
    ret = MPI_Error_string(var_0, var_1, var_2);
    ret = PMPI_Error_string(var_0, var_1, var_2);
    return 0;
}
